about.html

```diff
@@ -0,0 +1,223 @@
+              </div>
+            </nav>
+          </header>
+          <div class="intro about-wrap">
+            <p id="introText" class="hide">"Hello! Sharing World"</p>
+            <h1 class="heading mb-4 me-3">
+              Fund Raising
+              <br />
+              Concert in Myanmar 2023
+            </h1>
+          </div>
+          <div class="mb-lg-5 detail-wrap about-wrap">
+            <p class="detail detail-about">
+              The concert brings together the community of activists, experts
+              and all the people around the world. This year, we hope to expand
+              our invitation list including organizations and individuals
+            </p>
+            <p class="contact contact-about">
+              Please contact us per Email for any further questions about the
+              Fund Raising concert 2023!
+              <br />
+              <a href="#" class="fundLink">fundraising2023@icmyanmar.org</a>
+            </p>
+          </div>
+        </section>
+        <section class="mb-5">
+          <div class="p-3 text-center concert">
+            <div class="p-4">
+              <h2>
+                Iron Cross Fund Raising Concert 2023 Logo
+              </h2>
+              <div class="d-flex align-items-center justify-content-center">
+                <hr class="line" />
+              </div>
+              <p>
+                The logo of Iron Cross Fund Raising concert 2023 was decided
+                through the logo competition from 8.June to 7.July.
+              </p>
+            </div>
+            <div class="cross-wrap">
+              <div class="border mt-2 iron-cross">
+                <img src="./img/logo.png" alt="Iron Cross Logo" />
+              </div>
+            </div>
+          </div>
+          <div class="about-past">
+            <h2 class="text-center">See the past Iron Cross Concerts</h2>
+            <div class="d-flex align-items-center justify-content-center">
+              <hr class="line" />
+            </div>
+            <p class="text-center me-3 ms-3">
+              Take a look at the last two Iron Cross Concerts which took place
+              in Yangon and in Mandalay
+            </p>
+          </div>
+          <div class="concert-wrap">
+            <div class="concert-wrapper1 ms-3">
+              <div
+                class="d-flex flex-column align-items-center justify-content-center"
+              >
+                <p class="d-block h1">2020</p>
+                <p class="d-block">Fundraising Concert in Yangon</p>
+              </div>
+            </div>
+            <div class="concert-wrapper2 ms-3">
+              <div
+                class="d-flex flex-column align-items-center justify-content-center"
+              >
+                <p class="d-block h1">2021</p>
+                <p class="d-block">Fundraising Concert in Mandalay</p>
+              </div>
+            </div>
+          </div>
+        </section>
```

Comment:
- [ ] The `see past concerts` section should be a section on it own and not nested into another section for proper page structure. Kindly make it a separate section within the `body` tag.